cgi/generate_sample_import_file.pl

```diff
@@ -209,8 +189,16 @@
 		}
 
 		# Write cell and comment
+		if($fields_param && $fields_param eq 'mandatory'){
```

Comment:
You can simplify this block by calling "next;" if the script was called with &fields=mandatory is $importance is not "mandatory"'. That way you can leave $worksheet->write($headers_row, $col, $field_ref->{text}, $formats{$importance}); out of the block.